mods/cnc/maps/cnc64gdi01/cnc64gdi01.lua

```diff
@@ -176,7 +176,16 @@ end
 
 HeliHunt = function()
 	local helicopters = enemy.GetActorsByType("heli")
-	SquadHunt(helicopters)
+	local patrolpath = Utils.Random({
```

Comment:
Can we get that extracted to a global variable at the beginning of the script? Like
```
HeliPatrolPaths =
{
	{ HeliPatrol1.Location, HeliPatrol2.Location, HeliPatrol3.Location, HeliPatrol4.Location, HeliPatrol5.Location, HeliPatrol6.Location },
	{ HeliPatrol5.Location, HeliPatrol4.Location, HeliPatrol3.Location, HeliPatrol2.Location, HeliPatrol1.Location, HeliPatrol6.Location } 
}
```
Then just use `local patrolpath = Utils.Random(HeliPatrolPaths)` here.